metadata-ingestion/source_docs/athena.md

```diff
@@ -0,0 +1,62 @@
+# AWS Athena
+
+To install this plugin, run `pip install 'acryl-datahub[athena]'`.
+
+This plugin extracts the following:
+
+- List of databases and tables
```

Comment:
Not really, right? It really just extracts each table and binds it to a DataHub Dataset. Let's try to tighten some of these statements up. 

We may also want to consider a "validation" section below, which tells you how to verify that the source has worked